tests/unit/woopay/test-class-woopay-adapted-extensions.php

```diff
@@ -139,4 +139,24 @@ public function test_get_gift_cards_data_while_logged_out_with_zero_balance() {
 
 		$this->assertEquals( $this->woopay_adapted_extensions->get_gift_cards_data( $this->test_user ), $expected );
 	}
+
+	public function test_get_extension_data() {
```

Comment:
Should we update the option `'woocommerce_currency'` to USD in case it's not USD so that we won't break the test in the future if the currency changes?

Reply:
I've tried using these codes but no success changing the currency during the tests:

```
add_filter(
	'pre_option_woocommerce_currency',
	function () {
		return 'BRL';
	}
);

add_filter(
	'woocommerce_currency',
	function () {
		return 'BRL';
	}
);

update_option( 'woocommerce_currency', 'BRL' );
```